src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -1334,6 +1338,72 @@ enum CandidateStatus {
     EXCLUDED,
   }
 
+  /**
+   * A wrapper around Map&lt;Integer, RoundTally&gt;
+   * A mapping between a round number and that round's corresponding RoundTally.
+   */
+  static class RoundTallies {
+    private final Map<Integer, RoundTally> roundTallies = new HashMap<>();
+
+    public void put(Integer roundNumber, RoundTally roundTally) {
+      roundTallies.put(roundNumber, roundTally);
+    }
+
+    public RoundTally get(Integer roundNumber) {
+      return roundTallies.get(roundNumber);
+    }
+  }
+
+  /**
+   * A wrapper around Map&lt;TabulateByField, Map&lt;String, T&gt;&gt;
+   * Breaks down the templated type T by a specific field.
+   * The field type is the outer map (TabulateByField), and the field ID is the inner map (String).
+   */
+  static class BreakdownByField<T> {
```

Comment:
OK, so you made this generic so that you could reuse it for both tallies and tally transfers.